### PR TITLE
Adds "purgeCompletedSessions" job

### PR DESCRIPTION
**Pull request by rfornea**
-Adds "PurgeCompletedSessions" job and tests
-Moves completed data maps to a completed_data_maps table.  Will need this if we decide we care about confirmation rates, though we might not care anymore
-Deletes completed sessions

### CODE REVIEW

File: jobs/purge_completed_sessions.go [hunk 1]
```diff
@@ -1 +1,75 @@
...
+	for _, genesisHash := range genesisHashesNotComplete {
+		notComplete[genesisHash.GenesisHash] = true
+	}
+
+	var moveToComplete = []models.DataMap{}
+
+	for _, genesisHash := range allGenesisHashes {
+		if !notComplete[genesisHash] {
+			models.DB.RawQuery("SELECT * from data_maps WHERE genesis_hash = ?", genesisHash).All(&moveToComplete)
+			MoveToComplete(moveToComplete)
```
Comment (AaronVasquez) [inline jobs/purge_completed_sessions.go]: Wrap lines 48-50 in a transaction so we don't have inconsistent data.

File: models/completed_data_maps.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import (
+	"encoding/json"
+	"time"
+	"github.com/gobuffalo/pop"
+	"github.com/gobuffalo/validate"
+	"github.com/gobuffalo/validate/validators"
+)
+
+type CompletedDataMap struct {
+	ID          int       `json:"id" db:"id"`
```
Comment (AaronVasquez) [inline models/completed_data_maps.go]: You can merge fields from another struct like this:
```go
type CompletedDataMap struct {
    models.DataMap
}
```

Comment (rfornea) [reply models/completed_data_maps.go]: This doesn't seem to work, or at least I can't seem to make it work.  You can inherit from other structs, but the fields won't keep the same indentation level.  This demonstrates the issue:  https://play.golang.org/p/8VH15cNga9R  

Unless there is some other way to do it.

File: jobs/purge_completed_sessions.go (hunk 1 above)
Comment (rfornea) [reply jobs/purge_completed_sessions.go]: Okay

File: models/completed_data_maps.go (hunk 2 above)
Comment (AaronVasquez) [reply models/completed_data_maps.go]: ok, nvm then

**Pull request merged**